pyw3d/objects.py

```diff
@@ -252,8 +252,94 @@ def fromXML(content_root):
             return W3DLight.fromXML(content_root)
         if content_root.find("ParticleSystem") is not None:
             return W3DPSys.fromXML(content_root)
+        if content_root.find("Shape") is not None:
+            return W3DShape.fromXML(content_root)
         raise BadW3DXML("No known child node found in Content node")
 
+class W3DShape(W3DContent):
+    """ Create a shape object in virtual space
+
+    :param str shape_type: type of shape to be created
+    :param int radius: radius of the shape to be added
+    :param tuple color: Three floats representing RGB color of object 
+    :param int depth: height of shape, if shape is cone/cylinder
+    """
+    ui_order = ["shape_type","radius", "rotation", "depth"]
```

Comment:
Nice! I haven't been good about keeping these up to date, but I'm really glad you added this. It'll make my life easier later on.